app/controllers/cinema_halls_controller.rb

```diff
@@ -1,4 +1,76 @@
 # frozen_string_literal: true
 
 class CinemaHallsController < ApplicationController
+	def index 
+		@cinema_halls = Cinema_hall.all
```

Comment:
Did you run rubocop? I see that you have inconsistances in indentations 🤔

Reply:
I think it should be `CinemaHall.all` instead of `Cinema_hall` :)